### PR TITLE
feat: add quote component

### LINKED ISSUE #177: Component: Quote ContentBlock
### Design
https://www.figma.com/file/FWlI63N28LP7BxqNqj1bGw/sy-004-(Design!)?node-id=3701%3A23643

### PR DESCRIPTION
**Pull request by hofmann-felix**
Resolves: https://github.com/satellytes/satellytes.com/issues/177
Storybook: https://deploy-preview-216--satellytes-website-storybook.netlify.app

### What's included?
- new Quote content block

### Preview
Bildschirmfoto 2021-11-25 um 13 42 27

### CODE REVIEW

File: src/components/quote/quote.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  color: ${theme.palette.text.topline};
+  padding-left: 20px;
+  margin: 0;
+`;
+
+const Quote = ({ children }: QuoteProps) => {
+  return <QuoteText>{children}</QuoteText>;
+};
+
+export default Quote;
```
Comment (feedm3) [inline src/components/quote/quote.tsx]: Named exports are preferred over default exports. Some reasons why: https://stackoverflow.com/a/68665805/3141881

So please change it to `export const Quote`.

**Review by feedm3 — CHANGES_REQUESTED** [review]
Simple approach, like it :)

File: src/components/quote/quote.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+import { TextStyles } from '../typography/typography-v2';
+import { theme } from '../layout/theme';
+
+interface QuoteProps {
+  children: ReactNode | ReactNode[];
+}
+
+const QuoteText = styled.p`
```
Comment (georgiee) [inline src/components/quote/quote.tsx]: `blockquote` is the correct semantic tag to use for quotes. See here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote.

File: src/components/quote/quote.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  ${TextStyles.quote}
+  color: ${theme.palette.text.topline};
+  padding-left: 20px;
+  margin: 0;
+`;
+
+const Quote = ({ children }: QuoteProps) => {
+  return <QuoteText>{children}</QuoteText>;
```
Comment (georgiee) [inline src/components/quote/quote.tsx:18]: The design doesn't show any cite sources and such but the this [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote) shows a great example of how to do it with `figure` combined with the blockquote. Maybe we can work on that in a following PR to make this component more useful as I can imagine to use this in blog posts if available.

```
<figure>
    <blockquote cite="https://www.huxley.net/bnw/four.html">
        <p>Words can be like X-rays, if you use them properly—they’ll go through anything. You read and you’re pierced.</p>
    </blockquote>
    <figcaption>—Aldous Huxley, <cite>Brave New World</cite></figcaption>
</figure>
```

File: src/components/quote/quote.tsx (hunk 1 above)
Comment (georgiee) [reply src/components/quote/quote.tsx]: I fully support this. You almost never need default exports. Sometimes I try it out to challenge my thoughts about this but it's much more powerful sticking to the named exports.

File: src/components/quote/quote.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+<Meta title="Components/Quote" component={Quote} />
+
+# Quote
+
+Special text formatting for quotes. Used mainly in blog posts. Text is passed through `children`.
+
+<Canvas>
+  <Story story={Regular}/>
+</Canvas>
```
Comment (georgiee) [inline src/components/quote/quote.stories.mdx:13]: Thanks for taking care of writing at least a little bit 🙌

**Review by georgiee — CHANGES_REQUESTED** [review]
Thanks for that 👍 The wrong tag one is blocking for me otherwise only a few comments.

File: src/components/quote/quote.tsx (hunk 3 above)
Comment (hofmann-felix) [reply src/components/quote/quote.tsx:18]: Good suggestion, I created an issue (https://github.com/satellytes/satellytes.com/issues/221) for it

Comment (georgiee) [reply src/components/quote/quote.tsx:18]: Thanks 👍

File: src/components/quote/quote.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+import { TextStyles } from '../typography/typography-v2';
+import { theme } from '../layout/theme';
+
+interface QuoteProps {
+  children: ReactNode | ReactNode[];
+}
+
+const QuoteText = styled.blockquote`
```
Comment (georgiee) [inline src/components/quote/quote.tsx:10]: Thanks for changing 👍

**Review by georgiee — APPROVED** [review]
Perfect, let's merge 💪
I have checked the design and found some invisible box shadow. I guess this is an error on the design but I asked for the intention. Too late for this PR, everything looks good now. Thank you.

**Review by georgiee — APPROVED** [review]
That's good now. Thanks for the correct tag change. Let's merge 👍

**Review by feedm3 — APPROVED** [review]

**Comment by georgiee** [comment]
@hofmann-felix we forgot the border to the left. That was the invisible shadow box.  Look at the left side to spot the border.

Screen Shot 2021-11-29 at 15 16 23

**Pull request merged**